lib/scrabble/player.rb

```diff
@@ -0,0 +1,51 @@
+module Scrabble
+
+	class Player
+		attr_accessor :plays, :tiles
+		attr_reader :name
+
+		def initialize(name)
+			@name 	= name
+			@plays 	= []
+			@tiles 	= []
+		end
+
+		def play(word)
+			# guard clause for if word is invalid
+			return "ERROR -- invalid input" unless Scrabble.valid_input?(word)
+			# guard clause for if player has already won
+			return false if won? == true
```

Comment:
`won?` is "truthy" already, so you don't need to compare it to `true`. `if won?` will give the same result.